aixplain/modules/model_response.py

```diff
@@ -0,0 +1,25 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class ModelResponse:
+    """Class for keeping track of an item in inventory."""
+
+    status: str
```

Comment:
Use typing to set parameter type.

References:
- https://docs.python.org/3/library/typing.html
- https://github.com/aixplain/agentification/blob/development/agentification/utilities/models.py#L6